charts/ceph-csi-rbd/values.yaml

```diff
@@ -219,16 +219,12 @@ topology:
 # Variables for 'internal' use please use with caution! #
 #########################################################
 
-# The directory where the provisioner and pluginSocketFile will be mounted
-socketDir: /var/lib/kubelet/plugins/rbd.csi.ceph.com
 # The filename of the provisioner socket
 provisionerSocketFile: csi-provisioner.sock
 # The filename of the plugin socket
 pluginSocketFile: csi.sock
-# The directory where nodeplugin socket is created
-registrationDir: /var/lib/kubelet/plugins_registry
-# The nodeplugin plugins directory
-pluginDir: /var/lib/kubelet/plugins
+# kubelet working directory,can be set using `--root-dir` when starting kubelet.
+kubeletDir: /var/lib/kubelet
```

Comment:
Maybe mention the previously used, but now removed variables?

Reply:
updated commit message for removed variables.

Reply:
It was more meant for a comment here. Uses that have these values in their configuration will not be able to find them anymore. On the other hand, these are marked as *internal*.